### PR TITLE
README Update

### PR DESCRIPTION
**Pull request by HuckMatt**
Update README to standard format

Add LISCENSE

Add .env

Update meeting.jsk to allow for high-security Agora projects

### CODE REVIEW

**Comment by HuckMatt** [comment]
@eric-giannini please review

File: README.md [hunk 1]
```diff
@@ -1,62 +1,103 @@
...
+## Introduction
 
-### Obtain Symbl App ID and Secret and Agora App ID
+This sample implementation shows you how to quickly create a live app using the Symbl Websocket adapter with Agora Web SDK.
 
-To use the Agora example application with Symbl, you'll need to:
-1. Create an account. You can signup at [platform.symbl.ai/#/signup](https://platform.symbl.ai/#/signup). 
-2. Head to the [dashboard](https://platform.symbl.ai/#/home)
-3. Save your App Id and Secret 
+## Pre-requisites
```
Comment (eric-giannini) [inline README.md]: It should be three pound signs.

File: README.md [hunk 2]
```diff
@@ -1,62 +1,103 @@
...
+    "react-scripts": "3.1.1",
+    "symbl-chime-adapter": "^1.0.9"
+  }
+```
 
-## License
+## Community
 
-The MIT License (MIT)
+If you liked our integration guide, please star our repo! If you have any questions, feel free to reach out to us at devrelations@symbl.ai or through our Community Slack at https://developer.symbl.ai/community/slack or our [developer community][developer_community].
```
Comment (eric-giannini) [inline README.md]: Reformat to display properly.

**Comment by HuckMatt** [comment]
@eric-giannini updated

**Comment by eric-giannini** [comment]
@HuckMatt Are those changes updated?

**Comment by HuckMatt** [comment]
@eric-giannini yes, my most recent commit fixed the display issues.

**Pull request merged**